extensions/mssql/src/objectManagement/ui/restoreDatabaseDialog.ts

```diff
@@ -53,6 +55,8 @@ export class RestoreDatabaseDialog extends ObjectManagementDialogBase<Database,
 	private logFileFolder: azdata.InputBoxComponent;
 	private logFileFolderButton: azdata.ButtonComponent;
 	private logFileFolderContainer: azdata.FlexContainer;
+	private targetDatabase: azdata.DropDownComponent;
+	private isManaged: boolean;
```

Comment:
I'd suggest the full `isManagedInstance` to make it clear when this would be true